app/src/main/java/ch/epfl/sdp/GoogleMapApi.java

```diff
@@ -119,6 +119,9 @@ public Bitmap createSmallCircle(int color) {
 
     @Override
     synchronized public void displayEntity(Displayable displayable) {
+        if (displayable == null) {
+            return;
+        }

```

Comment:
Maybe it is better to throw an Exception

Reply:
The thing is that it can happen that we try to display a currentLocation which is null since it comes from the network. So just doing nothing is fine I would tell. (fail safe)